website/theme/i18n/zhCN.ts

```diff
@@ -1,40 +1,76 @@
 import type { EN_US } from './enUS';
 
 export const ZH_CN: Record<keyof typeof EN_US, string> = {
-  coldStart: '冷启动（dev）',
-  coldBuild: '冷构建',
-  hmr: '热更新',
-  guide: '指南',
-  quickStart: '快速开始',
-  features: '核心特性',
-  compatibility: 'webpack 兼容性',
-  migration: '迁移指南',
-  cli: 'CLI',
-  ecosystem: '生态',
-  community: '社区',
+  // hero
+  heroSlogan: '基于 Rust 的高性能 Web 打包工具',
+  heroSubSlogan: '使用兼容 API 无缝替换 webpack',
+  getStarted: '快速开始',
+  learnMore: '深入了解',
+
+  // whyRspack
+  whyRspack: '为什么使用 Rspack？',
```

Comment:
这里用 "什么是" 可能更合适，比较简短
```suggestion
  whyRspack: '什么是 Rspack？',
```